spec/cube_spec.rb

```diff
@@ -436,4 +436,171 @@
 
   end
 
+  describe "cache" do
```

Comment:
Better create a separate `cube_cache_control_spec.rb` file and move these tests to this separate file.